media/stl.ksy

```diff
@@ -45,14 +45,6 @@ types:
         type: vec3d
         repeat: expr
         repeat-expr: 3
-  vec3d:
-    seq:
-      - id: x
-        type: f4
-      - id: y
-        type: f4
-      - id: z
-        type: f4
       - id: abr
```

Comment:
This belongs to every vertex: https://github.com/FreeCAD/FreeCAD/blob/6cc1524a0ddbfbcec76b3ef4407aeb79b7fbdc57/src/Mod/Mesh/App/Core/MeshIO.cpp#L1315